vm/src/obj/objfunction.rs

```diff
@@ -15,6 +15,19 @@ pub fn init(context: &PyContext) {
         context.new_member_descriptor(function_code),
     );
 
+    let builtin_function_or_method_type = &context.builtin_function_or_method_type();
```

Comment:
I don't think we need to call the method here, you can just access the member directly: we just need a reference, we don't need to clone it. (See function_type above lacks parens.)

This pattern of having a method and member of the same name where one does a clone is confusing, sorry.

Reply:
I fixed this with 7e1589d.